CRM/Contribute/BAO/ContributionRecur.php

```diff
@@ -777,6 +777,13 @@ public static function recurringContribution(&$form) {
     CRM_Core_Form_Date::buildDateRange($form, 'contribution_recur_next_sched_contribution_date', 1, '_low', '_high', ts('From'), FALSE, FALSE, 'birth');
     CRM_Core_Form_Date::buildDateRange($form, 'contribution_recur_failure_retry_date', 1, '_low', '_high', ts('From'), FALSE, FALSE, 'birth');
     CRM_Core_Form_Date::buildDateRange($form, 'contribution_recur_cancel_date', 1, '_low', '_high', ts('From'), FALSE, FALSE, 'birth');
+
+    // Add field for contribution status
+    $contributionStatus = CRM_Contribute_PseudoConstant::contributionStatus();
+    $form->addSelect('contribution_recur_contribution_status_id',
+      array('entity' => 'contribution', 'multiple' => 'multiple', 'context' => 'search', 'options' => $contributionStatus)
```

Comment:
Can you directly used the function in ```options``` i.e.
```..., 'context' => 'search', 'options' => CRM_Contribute_PseudoConstant::contributionStatus()```
because $contributionStatus isn't used anywhere else.